src/stamps/base.ts

```diff
@@ -0,0 +1,64 @@
+import { PostageBatch } from '@ethersphere/bee-js'
+import { stampGetCounter, stampGetErrorCounter } from './counters'
+import { logger } from '../logger'
+import { ERROR_NO_STAMP, StampsConfig, StampsConfigAutobuy, StampsConfigExtends } from '../config'
+
+export interface StampsManager {
+  start?: (config: StampsConfig) => Promise<void>
+  stop: () => void
+  postageStamp: () => string
+}
+
+export class BaseStampManager implements StampsManager {
```

Comment:
I don't think the `BaseStampManager` really needs to implement the `StampsManager` interface, it is sufficient if the child classes will, but if you think it is better to have it here as well, then feel free to leave it here.